### PR TITLE
added german translation

### PR DESCRIPTION
**Pull request by torwart**

### CODE REVIEW

**Comment by JakeWharton** [comment]
Thanks! I'll merge this when I turn on lint (#13) sometime tomorrow or Monday. Want to make sure it's sanity checking translation strigs for me.

**Comment by torwart** [comment]
@JakeWharton No problem ;)

File: telecine/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  <string name="record">Aufnehmen</string>
+  <string name="countdown_one">1…</string>
+  <string name="countdown_two">2…</string>
+  <string name="countdown_three">3…</string>
+  <string name="launch_overlay">Overlay starten</string>
+  <string name="remove_task">Schließen und von zuletzt benutzten Apps entfernen</string>
+  <string name="Preferences">Einstellungen</string>
+  <string name="countdown">3-Sekunden Countdown</string>
+  <string name="recents_hide">in zuletzt benutzten Apps verstecken</string>
+  <string name="video_size_percentage">Videgröße</string>
```
Comment (aried3r) [inline telecine/src/main/res/values-de/strings.xml]: Missing an "o" here.

**Comment by torwart** [comment]
@aried3r Thanks for this. Have fixed it in https://github.com/JakeWharton/Telecine/commit/c55aa168080acab5b6d7498c3b392dadb643c615

**Comment by JakeWharton** [comment]
Squashed and merged. Thanks!

**Pull request closed without merging**